migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -93,7 +92,7 @@ If an application uses an internal image in the `openshift` namespace that is no
 * [ ] Certificates that your application uses exist on the target cluster.
 * [ ] You have configured appropriate firewall rules on the target cluster.
 * [ ] You have correctly configured load balancing on the target cluster.
-* [ ] If a namespace on the target cluster duplicates a namespace on the source cluster, the resources that already exist in the target namespace do not have the same names as the resources that are being migrated.
+* [ ] If a namespace already exists on the target cluster that has the same name as a namespace being migrated from the source, you have made sure that there are no objects in the target namespace that are of the same type and have the same name as objects being migrated.
```

Comment:
@sbeskin-redhat Based on Erik's explanation, I believe that this is fine. @eriknelson: Is this correct?